examples/tornjak/values-auth.yaml

```diff
@@ -0,0 +1,12 @@
+spire-server:
+  tornjak:
+    config:
+      ## @extra tornjak.config.userManagement [object] UserManagement config
+      userManagement:
+        issuer: "http://host.docker.internal:8080/realms/tornjak"
+        audience: "tornjak-backend"
+        
+tornjak-frontend:
+  auth:
+    enabled: true
+    authServerURL: http://localhost:8080/ # enable auth by providing url
```

Comment:
could you change authServerURL to serverURL, since its already in the auth section, it makes tornjak-frontend.auth.authServerURL a little redundant.